input/groups/armor.json

```diff
@@ -1197,7 +1161,7 @@
       ]
     },
     "Guild Wars 2: Heart of Thorns (HoT)": {
-      "Elite Specialization": [
+      "Elite Specialization Generation 1": [
```

Comment:
If we keep it for legendary weapons I think we should keep it for elite specializations as well.

Personally, I don't like having two groups with the same name under the same section, but that's just me :D.

Reply:
I think the difference to me is people don't refer to elite specialisation weapons as being gen 1/gen 2, but they do refer to legendary weapons that way. This is probably just because there's a single elite specialisation collection for each elite spec.